src/SSVOpenHexagon/Core/Joystick.cpp

```diff
@@ -134,6 +134,11 @@ enum class AxisDir : int
     Right = 1
 };
 
+inline AxisDir operator-(AxisDir dir)
```

Comment:
```suggestion
[[nodiscard]] AxisDir operator-(AxisDir dir)
```

No need for `inline` here, TBH.